hallux/proposals/simple_proposal.py

```diff
@@ -78,22 +85,25 @@ def try_fixing(self, query_backend: QueryBackend, diff_target: DiffTarget) -> bo
             "ISSUE_TYPE": self.issue.issue_type,
             "ISSUE_DESCRIPTION": self.issue.description,
             "ISSUE_FILEPATH": self.issue.filename,
-            "ISSUE_LINES": "\n".join(self.issue_lines),
+            "ISSUE_LINES": "".join(self.issue_lines),
         }
 
         user_message = user_message_template.format(**issue_data)
         query_results: list[str] = query_backend.query(user_message, self.issue, issue_lines=self.issue_lines)
         if len(query_results) == 0:
             return False
 
-        proposed_lines: list[str] = query_results[0].split("\n")
+        proposed_lines: list[str] = query_results[0].splitlines(keepends=True)
```

Comment:
```suggestion
        proposed_lines: list[str] = self._split_lines(query_results[0])
```
Do we need to use your new func??

Reply:
well, we need to rewrite unit tests to not use internal methods, just public APIs. I exposed this one to fix unit tests